src/routes/tasks/components/TaskInputsDrawerSection.tsx

```diff
@@ -251,26 +308,29 @@ const TaskInputSectionEntry = ({
     identifier,
     algoInput,
     inputs,
+    inputsAssets,
 }: {
     identifier: string;
-    algoInput: InputT;
+    algoInput: AlgoInputT;
     inputs: TaskInputT[];
+    inputsAssets: TaskIOT[];
 }): JSX.Element => {
     const icon = inputIcon(algoInput.kind);
     const title = identifier;
 
     if (algoInput.multiple || inputs.length > 1) {
-        //  '|| inputs.length > 1' is for resilience to inconsistent data
+        //  '|| Object.entries(inputs).length > 1' is for resilience to inconsistent data
```

Comment:
Is this change in the comment still relevant?

Reply:
Or maybe the check should be on Object.keys(input)

Reply:
No change in the comment is no longer relevant, nice catch. Left over from when i tried to pass inputs as an object.